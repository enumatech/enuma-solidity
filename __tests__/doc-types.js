const docTypes = require('../src/doc-types');

test('docTypes', () => {
  expect(docTypes).toMatchSnapshot();
});
