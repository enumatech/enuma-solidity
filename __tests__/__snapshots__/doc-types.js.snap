// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`docTypes 1`] = `
Object {
  "address": "string",
  "bool": "boolean",
  "int": "number",
  "int104": "number",
  "int112": "number",
  "int120": "number",
  "int128": "number",
  "int136": "number",
  "int144": "number",
  "int152": "number",
  "int16": "number",
  "int160": "number",
  "int168": "number",
  "int176": "number",
  "int184": "number",
  "int192": "number",
  "int200": "number",
  "int208": "number",
  "int216": "number",
  "int224": "number",
  "int232": "number",
  "int24": "number",
  "int240": "number",
  "int248": "number",
  "int256": "number",
  "int32": "number",
  "int40": "number",
  "int48": "number",
  "int56": "number",
  "int64": "number",
  "int72": "number",
  "int8": "number",
  "int80": "number",
  "int88": "number",
  "int96": "number",
  "string": "string",
  "uint": "number",
  "uint104": "number",
  "uint112": "number",
  "uint120": "number",
  "uint128": "number",
  "uint136": "number",
  "uint144": "number",
  "uint152": "number",
  "uint16": "number",
  "uint160": "number",
  "uint168": "number",
  "uint176": "number",
  "uint184": "number",
  "uint192": "number",
  "uint200": "number",
  "uint208": "number",
  "uint216": "number",
  "uint224": "number",
  "uint232": "number",
  "uint24": "number",
  "uint240": "number",
  "uint248": "number",
  "uint256": "number",
  "uint32": "number",
  "uint40": "number",
  "uint48": "number",
  "uint56": "number",
  "uint64": "number",
  "uint72": "number",
  "uint8": "number",
  "uint80": "number",
  "uint88": "number",
  "uint96": "number",
}
`;
